ig.ini

```
@@ -1,50 +1,50 @@
-[IG]
-# ini file for the Implementation Guide publisher
-# see comments below for instructions
-
-ig = fsh-generated/resources/ImplementationGuide-hl7.fhir.fr.preadmission.json
-template = https://github.com/Interop-Sante/hl7.ig.fr.template
-#template = #local-template
-
-
-
-##########################
-### ig.ini parameters: ###
-##########################
-
-#   ig: the name of the implementation guide resource file.
-examples:
-# ig = input/myig.xml
-# ig = input/implementationguide-example2.xml
-
-#   template: the IG template that will be used to create the implementation guide (normally be a package name, but can be a local folder).
-#   base fhir template:
-#template = fhir.base.template
-
-#    HL7 (non-FHIR) IG template:
-#template = hl7.base.template
-
-#   HL7 FHIR template:
-#template = hl7.fhir.template
-
-#   it's possible to specify a specific version of the template
-#template = fhir.base.template#0.1.0
-#   or the current version:
-#template = fhir.base.template#current
-#   if no version is specified, the publisher will take the #current version
-
-#   local templates can be used by prefixing a relative path with # :
-#template = #mylocaltemplate
-#template = #..\templates\mytesttemplate1
-
-#   local templates can also use absolute paths:
-#template = #C:\MyWork\ImplementationGuides\mytemplates\mytesttemplate1
-
-
-#   other parameters are defined in the ImplementationGuide resource:
-#   https://confluence.hl7.org/display/FHIR/Implementation+Guide+Parameters
-
-#   for more documentation on implementation guides and templates, see the FHIR Guidance ImplementationGuide 
-#   http://build.fhir.org/ig/FHIR/ig-guidance
-
-
+[IG]
+# ini file for the Implementation Guide publisher
+# see comments below for instructions
+
+ig = fsh-generated/resources/ImplementationGuide-hl7.fhir.fr.preadmission.json
+template = https://github.com/Interop-Sante/hl7.ig.fr.template

```

```suggestion
template = https://github.com/Interop-Sante/hl7.ig.fr.template/tree/template-IS-colors
```

Pour tester le nouveau template InteropSanté si ça te tente!